src/OpenLoco/Vehicles/Vehicle2.cpp

```diff
@@ -1,16 +1,353 @@
+#include "../CompanyManager.h"
 #include "../Interop/Interop.hpp"
+#include "../MessageManager.h"
+#include "../Objects/RoadObject.h"
+#include "../Objects/TrackObject.h"
+#include "../Tutorial.h"
 #include "Vehicle.h"
 
 using namespace OpenLoco::Interop;
+using namespace OpenLoco::Literals;
 
 namespace OpenLoco::Vehicles
 {
+    static loco_global<uint32_t, 0x01136114> vehicleUpdate_var_1136114;
+    static loco_global<int32_t, 0x0113612C> vehicleUpdate_var_113612C; // Speed
+    static loco_global<int32_t, 0x01136130> vehicleUpdate_var_1136130; // Speed
+    static loco_global<Speed32, 0x01136134> vehicleUpdate_var_1136134; // Speed
+    static loco_global<VehicleHead*, 0x01136118> vehicleUpdate_head;
+
+    // values are pre *256 for maths
+    static constexpr std::array<int32_t, 13> _500170{
```

Comment:
what are these values for?

Reply:
I'm not entirely sure but think they are something to do with amount of negative force due to the angle of the track and gravity.
So its like `256 * 256 * SIN(angle)`

Reply:
Interesting. There is a strange pattern as well:
- ignore first zero
- 4 values in position 1->5
- next four values are the negative of the first four
- next 2 values are negatives of each other
- last 2 values also negatives of each other

Reply:
ah so it does, that's useful to know, maybe we should add a comment mentioning these array values follow the sprite pitches?